### PR TITLE
🎃 Add pumpkin support

### PR DESCRIPTION
**Pull request by seaweeddol**
Spooky pumpkins 🎃

### CODE REVIEW

**Comment by carolstran** [comment]
@seaweeddol, I absolutely love this! It's the perfect addition for spooky season 🎃 

Before merging, could you please make sure everything is in alphabetical order? I'll comment on which ones I mean.

File: sproutify.js [hunk 1]
```diff
@@ -78,6 +78,20 @@ const tomatoImagePaths = [
   "images/tomatoes/tom-hermans-nM6qrtnVKn8-unsplash.jpg"
 ];
 
+const pumpkinImagePaths = [
```
Comment (carolstran) [inline sproutify.js:71]: Could you move this variable above `tomatoImagePaths`?

File: sproutify.js [hunk 2]
```diff
@@ -86,7 +100,8 @@ const map = {
   candyCorn: candyCornImagePaths,
   eggplants: eggplantImagePaths,
   potatoes: potatoImagePaths,
-  tomatoes: tomatoImagePaths
+  tomatoes: tomatoImagePaths,
+  pumpkins: pumpkinImagePaths
```
Comment (carolstran) [inline sproutify.js]: This one too, please put it above the `tomatoes` key so it's in alphabetical order 😊

File: veggie-selector/options.html [hunk 3]
```diff
@@ -18,6 +18,7 @@
         <option value="eggplants" class="option"> &#127814; eggplants</option>
         <option value="potatoes" class="option">&#129364; potatoes</option>
         <option value="tomatoes" class="option">&#127813; tomatoes</option>
+        <option value="pumpkins" class="option">&#127875; pumpkins</option>
```
Comment (carolstran) [inline veggie-selector/options.html:20]: Same as the other comments, please move above tomatoes ⬆️  🍅

**Review by carolstran — CHANGES_REQUESTED** [review]
Commented to show which ones should be moved so it's in alphabetical order. Small thing I know, but it helps keep everything in order 😂 

Thank you!!

**Comment by carolstran** [comment]
I also updated the name to include an emoji 🎃 🎃 🎃

File: veggie-selector/options.html (hunk 3 above)
Comment (seaweeddol) [reply veggie-selector/options.html:20]: Moved all the pumpkins above the tomatoes! 🎃

**Review by carolstran — APPROVED** [review]
Amazing! Thank you! 🎃  💯  🎉  I will release a new version of the add-on early this week and let you know.

**Pull request merged**